tests/test_envs.py

```diff
@@ -56,6 +56,17 @@ def test_atari_space_invaders():
     assert len(space_invader_environments) > 0
 
 
+def test_no_atari_unmasked():
+    """Tests that we only load Atari envs with score masking implemented."""
+    non_masked_environments = list(
+        filter(
+            lambda name: _get_score_region(name) is None,
+            ATARI_ENVS,
+        ),
+    )
+    assert len(non_masked_environments) == 0
```

Comment:
So I guess now we're forcing all environments to be masked? Is this what we want? I don't have enough information to have a strong opinion on this, but intuitively it seems useful that a library providing algorithm-agnostic environment implementations should not enforce design choices only relevant to certain algorithms.

Reply:
Unless I'm missing something important, I would probably favor having both the masked and unmasked environments. The test you could add then is make sure there is always one of each, even though this would force anyone adding a new environment to also add the masks, but this is probably fine?

Reply:
Agree we want to still support the non-masked variants.